docs/pages/admin-guides/deploy-a-cluster/deployments/gcp.mdx

```diff
@@ -232,9 +232,6 @@ teleport:
   nodename: teleport-auth-server
   data_dir: /var/lib/teleport
   pid_file: /run/teleport.pid
-  connection_limits:
-    max_connections: 15000
```

Comment:
Why are we removing the `max_connections` as well?

Reply:
Because it's not consequential for this guide (a basic getting started on GCP guide)